src/components/NurseryWithdrawals/NurseryWithdrawalsFiltersPopover.tsx

```diff
@@ -95,6 +102,16 @@ export default function NurseryWithdrawalsFiltersPopover({
     setTemporalRecord(filters);
   }, [filters, setTemporalRecord]);
 
+  useEffect(() => {
+    const populatePlantingSites = async () => {
+      const result = await listPlantingSites(organization.id);
+      if (result.requestSucceeded) {
+        setPlantingSites(result.sites);
```

Comment:
else error?

Reply:
Same thing here, what message could we show that would not generate confussion?

Reply:
if user searches but the results don’t update because there was a silent error.. that would be confusing as well. If user searches and there’s an error and we show an error .. not sure that would be confusing? A generic error is better than silent errors

Reply:
Okay maybe I'm not understanding the flow too well.
Let's just iterate on this.
We don't have to complete the entire filtering in one PR, not easy to review.